src/templates/base/2019/contributors.html

```diff
@@ -293,11 +296,11 @@ <h1 class="title title-lg">
     </form>
 
     <ul class="contributor-grid">
-      <li>
-        <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/README.md" class="contributor-join" aria-labelledby="jointheteam">
-          <svg xmlns="http://www.w3.org/2000/svg" role="img" width="174" height="100" viewBox="0 0 203 117">
+      <li class="contributor-join">
+        <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/README.md" aria-labelledby="jointheteam">
+          <svg xmlns="http://www.w3.org/2000/svg" role="img" width="174" height="100" viewBox="0 0 173 96">
             <title id="jointheteam">{{ self.join_the_team_title() }}</title>
-            <path d="M0 0l11 89 71 4.4L65.9 117l40.7-22.4L201 89l2-79z" />
+            <path d="M0,0l8.4,75.1l88.7,0.8L93.3,96l24.9-19.1l50.6-1.8l3.7-67.5L0,0z"/>
```

Comment:
How do I learn this power? 😁

Reply:
I'm not a Jedi, I'm not doing this path edits manually although I'd wish 😁

Reply:
I did for #1069 for the cover version of the header SVG! Am sure it would have been possible to reuse the existing one but after fiddling for ages I resorted to manually guessing which parts of SVG code to tweak until I (eventually!) got what I wanted. Did reuse some @catalinred magic SVG code for the spine So slowly learning from him but a while’s to go yet...